.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      bump-type:
+        type: choice
+        description: The position of the version to be bumped.
+        options:
+        - patch
+        - minor
+        - major
+      tag_to_move:
+        type: choice
+        description: The generic tag to be advanced.
+        options:
+          - v1
```

Comment:
Options to select which part of the version to bump, and the generic tag to move on running this job.